application/src/test/java/org/togetherjava/tjbot/jda/JdaTester.java

```diff
@@ -487,6 +495,110 @@ public JdaTester() {
         return ErrorResponseException.create(reason, new Response(null, -1, "", -1, Set.of()));
     }
 
+    /**
+     * Creates a MessageReceivedEvent
+     * <p>
+     * The method creates a MessageReceivedEvent from the input message and mock the message author
+     * from the input user.
+     *
+     * @param message the message to create an event for
+     * @param user the author from the message
+     * @return the created MessageReceivedEvent
```

Comment:
* the message to receive
* the author who send the message
* the event of receiving the message